site/components/pages/Pledge/PledgeForm/index.tsx

```diff
@@ -86,6 +87,7 @@ export const PledgeForm: FC<Props> = (props) => {
         pageAddress: props.pageAddress,
         pledge: values,
         signature,
+        urlPath: asPath,
```

Comment:
Did you test paths with query params or hash routes? `asPath` includes both, might be safer to use `pathname`

But on second thought we don't even need to pass the URL, it's always `/pledge/${pageAddress}` right?
Is `pageAddress` the kns/ens domain or is it always the 0x address?

Reply:
Update: `pathname` gives us `/pledge/[address]`. So I went with constructing the url path using `query`